pkg/services/sqlstore/migrations/datasource_mig.go

```diff
@@ -138,4 +138,8 @@ func addDataSourceMigration(mg *Migrator) {
 	mg.AddMigration("Add is_prunable column", NewAddColumnMigration(tableV2, &Column{
 		Name: "is_prunable", Type: DB_Bool, Nullable: true, Default: "0",
 	}))
+
+	mg.AddMigration("Add api_version column", NewAddColumnMigration(tableV2, &Column{
+		Name: "api_version", Type: DB_Varchar, Nullable: true, Length: 40,
```

Comment:
Nit. Length could perhaps be adjusted? v0alpha99 is 9 characters so 10 or 11 _should_ be enough :)

Reply:
I followed the same approach than the UID but I guess we can make it shorter. 10 may be too tight, maybe 20?